src/org/opensolaris/opengrok/analysis/JFlexXref.java

```diff
@@ -182,7 +182,7 @@ public void reInit(char[] contents, int length) {
      *
      * @param reader new reader for this lexer
      */
-    public final void reInit(Reader reader) {
+    public void reInit(Reader reader) {
```

Comment:
I have a bad feeling this breaks how lucene analyzers are working, but I will have a look and confirm ... see below

Reply:
I believe the JFlexXref subclasses are not used with Lucene — only the JFlexTokenizer subclasses.